pkg/minikube/constants/constants.go

```diff
@@ -121,8 +121,10 @@ const (
 	DefaultKeepContext = false
 	// SHASuffix is the suffix of a SHA-256 checksum file
 	SHASuffix = ".sha256"
-	// DefaultMemory is the default memory of a host, in megabytes
-	DefaultMemory = 2048
+	// DefaultMemory is the default memory which will be allocated to minikube, parseable
+	DefaultMemory = "2048mb"
+	//MinimumMemorySizeMB is the minimum memory size, in megabytes
+	MinimumMemorySizeMB = 1500
```

Comment:
on this 1500 mb , could you check what is the absolute min way to start  minikube  ? I rather we allow extreme lows for people who wanna do pressure test kuberentes under low ram. I rather we have a hard fail on anything less than 1000 mb and a warnning for anything less than the default memory ( 2048mb) could you add a warning  and lower the MinimumMemorySizeMB?

Reply:
Kubernetes itself runs OK with 512M, but our minikube distribution probably requires 1024M

1500 seems a bit arbitrary, but either 512 ("safe") or 1024 (the old default) would do just fine

Reply:
I would be down for a hard fail for anything less than 512 mb and a warning for anything less than 2048 mb

Reply:
Could go with 1024 (or 1000?), seems like the current image doesn't even boot with 512...

Reply:
Sure. @tstromberg  mentioned the lowest he tried was 1.2 gb though